samples/Monitoring/deploy-mmaextension-windows-tags-loganalytics/azurepolicy.json

```diff
@@ -0,0 +1,140 @@
+{
+    "properties": {
+        "displayName": "Deploy Microsoft Monitoring Agent on Windows VMs based on a Tag",
+        "mode": "Indexed",
+        "description": "Deploy Microsoft Monitoring Agent on Windows VMs based on a Tag with value: MMA and Name: Windows",
```

Comment:
>Tag with value: MMA and Name: Windows [](start = 84, length = 37)

this is still not accurate.  User provides the tag name/value as parameters

Reply:
I missed this, I've updated it.